common/laws/00_health_system.txt

```diff
@@ -99,7 +99,7 @@ law_private_health_insurance = {
 	}
 	
 	modifier = {
-		country_bureaucracy_investment_cost_factor_mult = -0.1
+		country_institution_cost_institution_health_system_mult = 0.1
```

Comment:
you can also just leave it 0?

Reply:
I suppose it would be unnecessary to have a 10% increase yea